install/installer/pkg/components/ide-service/ide_config_configmap.go

```diff
@@ -110,6 +110,18 @@ func ideConfigConfigmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 					ImageLayers:       []string{codeWebExtensionImage, codeHelperImage},
 					LatestImage:       resolveLatestImage(ide.CodeIDEImage, "nightly", ctx.VersionManifest.Components.Workspace.CodeImage),
 					LatestImageLayers: []string{codeWebExtensionImage, codeHelperImage},
+					Versions: []ide_config.IDEVersion{
+						{
+							Version:     "1.87.0",
+							Image:       ctx.ImageName(ctx.Config.Repository, ide.CodeIDEImage, "commit-82dc424633bdc7266b46302042dd98af201fa8f8"),
+							ImageLayers: []string{codeWebExtensionImage, codeHelperImage},
```

Comment:
@jeanp413 just a question it means we evolve both web extension and code helper always in backward compatible manner, never bumping up engine and so on?

Reply:
> web extension and code helper always in backward compatible manner

good catch, I prefer to harcode the layers too then